homeassistant/components/samsungtv/strings.json

```diff
@@ -6,20 +6,19 @@
         "description": "Enter your Samsung TV information. If you never connected Home Assistant before you should see a popup on your TV asking for authorization.",
         "data": {
           "host": "[%key:common::config_flow::data::host%]",
-          "name": "[%key:common::config_flow::data::name%]"
+          "name": "Name"
         }
       },
       "confirm": {
-        "title": "Samsung TV",
-        "description": "Do you want to set up Samsung TV {model}? If you never connected Home Assistant before you should see a popup on your TV asking for authorization. Manual configurations for this TV will be overwritten."
+        "description": "Do you want to set up Samsung TV {model}? If you never connected Home Assistant before you should see a popup on your TV asking for authorization."
       }
     },
     "abort": {
-      "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
+      "already_in_progress": "Samsung TV configuration is already in progress.",
```

Comment:
Any reason why the standard string isn't being used?

Reply:
Fixed